test/run

```diff
@@ -22,6 +23,21 @@ import subprocess
 import sys
 import uuid
 
+DESCRIPTION = "Execute e2e and unit tests for Litani"
+
+
+def get_args():
+    pars = argparse.ArgumentParser(description=DESCRIPTION)
+    for arg in [{
+            "flags": ["--output-dir"],
+            "help": "output dir for test results",
+            "default": pathlib.Path(__file__).resolve().parent / "output",
+            "type": pathlib.Path
```

Comment:
🍄 This is going to give inconsistent results because the user might pass a relative path, but the default is an absolute path. I think we should `resolve` whatever the user gives us.

Thus, instead of `"type": pathlib.Path`, could you write a function that takes a string and returns a resolved path? Then you would do `"type": resolved_path`. Again, check out the 'type' documentation to see another example.

Reply:
I resolve what user gives us on line `167` - 
```
output_dir = args.output_dir.resolve()
```

Reply:
```
ronakf@88665a374b3f litani % ./test/run --output-dir /Users/ronakf/Desktop/litani/tmp/litani-results
Report will be rendered at file:///Users/ronakf/Desktop/litani/tmp/litani-results/latest/html/index.html
89 / 89 tests added
[24/93] no_timed_out_timeout_ignored:
```